modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -190,6 +190,7 @@ public Error OpenInExternalEditor(Script script, int line, int col)
                 case ExternalEditorId.CustomEditor:
                 {
                     string file = ProjectSettings.GlobalizePath(script.ResourcePath);
+                    string project = ProjectSettings.GlobalizePath("res://");
```

Comment:
I'm not entirely sure, removing them would make the behavior consistent with the other external editor flags, (since GlobalizePath replaces just "res:/" with its inner resource_path variable, which is used directly on the implementation of the other one). As is, this would work normally to tell a given external editor to open the project folder (I don't know any editor that would complain about a trailing slash on a directory path), but would make the behavior inconsistent when trying to open something within that folder (for example, if one specifies {project}/project.sln to attempt to open the solution file directly, this would result in 2 slashes back to back. However, I'm not sure if having them would cause any issues, or even if specifying paths relative to {project} would be a common practice, since this is an editor setting, and the contents of the project directory would change on a per project basis. Of course, I can modify this PR to remove the last character of the string if requested, to keep the same behavior as the other external editor setting.

Reply:
I also suspect that having a trailing slash or double slashes won't cause any issues, but I haven't tested this.

Reply:
Ok, I think it would be safer to remove it, so both external editor options behave the same. It should be possible to just remove the last character, without having to even check, since in the input string "res://", only "res:/" is replaced, and the last "/" remains there. I'll push a commit for this soon

Reply:
Done, tested on both VSCode and Kate on manjaro linux, by checking both behavior and the command line argument used to start the editors, and it seems to work as expected.